doc/changes/changes_0.1.0.md

```diff
@@ -0,0 +1,213 @@
+* fixed issue #22 - thanks abudis
+* moved dplyr methods to exasol-dplyr package & fixed bug: datatype def varchar NA
+* not suggests dplyr
+* bugfix dbReadTable schema handling
+* Fix typo in connection error hint
+* fixed issuestats badges, added example
+* bugfix dbReadTable schema handling
+* Fixed compilation warning
+* Fix Issue 37: Allow comments in the beginning of Queries (#39)
+* Fix start-test-env call in .travis.yaml (#44)
+* Fix CI config (#46)
+* Fix schema filter for dbListTables (#51)
+* #40: Fixed empty char column data type mapping (#55)
+* Removed broken badges
+
+## Documentation
+
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* moved exa.writeData to a separate file and added documentation as a doc-comment that is parsed by roxygen.
+* moved exa.readData to a separate file and added documentation as a doc-comment that is parsed by roxygen.
+* moved exa.createScript to a separate file and added documentation as a doc-comment that is parsed by roxygen.
+* enabled automated generation of NAMESPACE using roxygen
+* In order to pass R CMD CHECK, I added dummy comments for the following C functions:  - `C_asyncRODBCQueryStart`  - `C_asyncRODBCIOStart`  - `C_asyncRODBCIsDone`  - `C_asyncRODBCMax`  - `C_asyncRODBCProxyHost`  - `C_asyncRODBCProxyPort`  - `C_asyncRODBCQueryCheck`  - `C_asyncRODBCQueryFinish`
+* Added `\dontrun` for the examples to pass R CMD CHECK
+* Regenerated documentation using newer roxygen version.
+* Fixed some formatting in `README.md`
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* Update README.md
+* Added project status
+* Update README.md
+* Update README.md
+* Couple improvements in README to help getting started
+* Remove mention of AWS
+* update readme installation section
+* update readme installation section
+* readme changes
+* readme changes
+* readme changes
+* using new roxygen version
+* README clean-up
+* Update README.md
+* Update supported R Versions in README (#45)
+
+## Refactoring
+
+* Changesthat will later allow unit-testing
+* minor coding style fixes
+* Moved the example from `exa.createScript.R` to a separate `examples/createScript.R` file
+* Added `mockOnly` parameter to `exa.createScript` in order to support unit-testing. I also increased R package version number.
+* Reformatted C code in `exasol.c` for better understandability.
+* Added apt dependency `unixodbc-dev` that provides `sql.h`
+* unixodbc-dev used in travis config file
+* added `.travis.yml` to `.Rbuildignore` to pass R CMD CHECK
+* enabled "dontrun" within the createScript example
+* Added 2x badges : Travis-CI + Codecov
+* Moved examples from `readData` and `writeData` to separate files in `examples/` directory
+* changed DBItest repo to marcelboldt/DBItest
+* removed /lib folder
+
+## Dependency updates
+
+### Compile Dependency Updates
+* `DBI`
```

Comment:
Use the format https://www.google.de/maps/place/Gastst%C3%A4tte+am+Marienberg/@49.4776971,11.0917007,17z/data=!4m5!3m4!1s0x479f57db0d75f0ad:0x53b8a712faa861dc!8m2!3d49.4776971!4d11.0917007

```
Added <dependency groupId>:<dependency artifactId>:<version>
Updated <dependency groupId>:<dependency artifactId>:<old version> to <new version>
Removed <dependency groupId>:<dependency artifactId>:<version>
```